spec/lib/vmdb/config_spec.rb

```diff
@@ -59,6 +59,33 @@
       expect(VMDB::Config.save_file(data, resource)).to be true
       expect(SettingsChange.count).to eq(1)
       expect(SettingsChange.first).to have_attributes(:key => '/api/token_ttl', :value => "1.day")
+      expect(SettingsChange.first.resource).to eq(resource)
+    end
+
+    it "saves zone config" do
```

Comment:
Do we really need to add this given that this (multiple resource targets) is tested in the mixin specs? ref: https://github.com/ManageIQ/manageiq/blob/1009681373bed6951f4e5bcc6a9daa9e50dcfcfd/spec/models/mixins/configuration_management_mixin_spec.rb